### PR TITLE
Xamarin.Forms solution added

### PR DESCRIPTION
**Pull request by bmourat**

### CODE REVIEW

File: QuickstartApp/QuickstartApp.Android/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+//      Revision
+//
+// You can specify all the values or you can default the Build and Revision Numbers 
+// by using the '*' as shown below:
+// [assembly: AssemblyVersion("1.0.*")]
+[assembly: AssemblyVersion("1.0.0.0")]
+[assembly: AssemblyFileVersion("1.0.0.0")]
+
+// Add some common permissions, these can be removed if not needed
+[assembly: UsesPermission(Android.Manifest.Permission.Internet)]
```
Comment (guperrot) [inline QuickstartApp/QuickstartApp.Android/Properties/AssemblyInfo.cs]: These are not needed as they are added by the manifest merger already.

File: QuickstartApp/QuickstartApp.Android/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+[assembly: UsesPermission(Android.Manifest.Permission.WriteExternalStorage)]
```
Comment (guperrot) [inline QuickstartApp/QuickstartApp.Android/Properties/AssemblyInfo.cs]: Why your app accessing external storage?

Comment (bmourat) [inline QuickstartApp/QuickstartApp.Android/Properties/AssemblyInfo.cs]: It's not) These permissions are generated during project creation by default. I'll remove them.

**Comment by guperrot** [comment]
@elamalani confirmed the quick start is meant to be using PCL and not shared project.

File: QuickstartApp/QuickstartApp.Android/Resources/Resource.Designer.cs [hunk 3]
```diff
@@ -0,0 +1,5936 @@
+#pragma warning disable 1591
+//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
```
Comment (guperrot) [inline QuickstartApp/QuickstartApp.Android/Resources/Resource.Designer.cs]: This file is auto generated and does not to be in git.
Plus the content will differ on Mac and will pollute git status as something changed.
Please ignore it, that's what we do in Mobile Center Xamarin SDK.

**Pull request closed without merging**